Sources/TuistCore/Analytics/SelectiveTestAnalytics.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+import Path
+
+public struct SelectiveTestsAnalytics: Equatable {
+    public let hashes: [AbsolutePath: [String: String]]
+    public let cacheItems: [AbsolutePath: [String: CacheItem]]
```

Comment:
I have a similar suggestion here regarding the schema of this struct. Can we consolidate the two attributes if there's overlap?